less/mixins.less

```diff
@@ -1 +1,15 @@
 // Mixins below variables
+.flex-and-wrap {
```

Comment:
Nice mixin to add! Another good one for something so frequently used is a `.center()` mixin to handle display, justify-content, and align-items.